refpolicy: modify selinuxutil.te

We need to be able to create and delete links under
/etc/selinux/<policy>.  manage_lnk_file_perms is
probably more than we need, but it is a safe choice.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>


--- a/policy/modules/system/selinuxutil.te
+++ b/policy/modules/system/selinuxutil.te
@@ -450,6 +450,8 @@ allow semanage_t policy_config_t:file rw
 
 allow semanage_t semanage_tmp_t:dir manage_dir_perms;
 allow semanage_t semanage_tmp_t:file manage_file_perms;
+allow semanage_t selinux_config_t:lnk_file manage_lnk_file_perms;
+
 files_tmp_filetrans(semanage_t, semanage_tmp_t, { file dir })
 
 manage_dirs_pattern(semanage_t, semanage_var_lib_t, semanage_var_lib_t)
